scss/standard/_bootswatch.scss

```diff
@@ -20,3 +20,7 @@
 .pagination {
   margin: .5rem;
 }
+
+.btn.btn-link {
+  padding: 0 0 0 0;
```

Comment:
padding: 0 should achieve the same thing

Reply:
actually the build already breaks
Property `padding` should be written more concisely as `0` instead of `0 0 0 0`  shorthand-values

Reply:
Ah, didn't see that build failure for some reason. It's been fixed.

Reply:
checking out the review, might make sense to also add vertical-align: baseline? since btn sets it to middle, this undoes that and makes it normal text. Do you think that works to apply it everywhere?